samples/cds-sample-application/resources/service-specs/service-mapping.json

```diff
@@ -0,0 +1,7 @@
+{
+  "API_BUSINESS_PARTNER": {
```

Comment:
[q] Why did you include just on bp service? I know it was like this in the past but perhaps it is clearer to have on for cloud and OP? If you want to keep it simeple also fine but then remove the one line form the markdown about the second service.

Reply:
You're correct, I only kept the cloud because we originally only used the cloud vdm, and I thought adding both might possibly be confusing. The readme is already adjusted. See [here](https://github.com/SAP-samples/cloud-sdk-js/pull/386/files#diff-ca77b79e31045a26578e1d3040dad5c866609d4875d71c69668dd109348d7259)